pitch.md

```diff
@@ -1,44 +1,80 @@
-# Project Title (replace me)
+# Shared Spaces
 
 ## Overview
 _What's this project about, briefly?_
 
+This project serves a group of people who share a set of spaces and need to use the spaces at various times. Examples include kitchens, lab spaces, and study room. This language would ideally be good for any user to reserve a space or designate the space as 'occupied'.
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
-
 ### What's the need?
-_What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
-them?_
+_What need is met by your idea? Who are you helping? What is that person's experience like now? What would their experience be like if you could help them?_
 
+Shared spaces are often managed by some form of scheduler but the interfaces we use for them are nonintuitive and rather poor. This could help anyone who shares a resource with others, such as MuddVans, conference rooms, and study rooms. One example is a HMC student trying to reserve the Green Room. The student has to submit reservation details before being able to see the schedule. Then, they can only see 12 hours intervals of the schedule and cannot alter their original reservation without creating an entirely new query. This makes deciding on dates and times for meetings more difficult than it should be. If I could help them, the user would be able to query schedules in different ways (e.g. "Show me the schedule on November 28th for rooms that can hold 20 people"; "Reserve the Green room from 8PM to 9PM on November 28th"). There would also be ways to describe a new space that can also be reserved ("New Blue Room in Platt").
```

Comment:
I like the idea of having the room definition system mesh well with the room reservation system.